src/main/java/racingcar/vo/CarListFactory.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.vo;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.model.Car;
+
+public class CarListFactory {
+    private static final String DELIMITER_CAR_NAMES = ",";
+
+    public static CarList create(final List<Car> cars) {
+        final CarList carList = new CarList(cars);
+        return carList;
+    }
+
+    public static CarList create(final String carNames) {
+        final List<String> carNameList = getCarNameListFromCarNames(carNames);
+        final List<Car> carList = getCarList(carNameList);
+        final CarList carListObject = new CarList(carList);
+        return carListObject;
+    }
+
+    public static List<String> getCarNameListFromCarNames(final String carNames) {
+        final String[] carNamesSplittingByDelimiter = carNames.split(DELIMITER_CAR_NAMES);
+        final List<String> carNameList = Arrays.asList(carNamesSplittingByDelimiter)
+                .stream()
+                .map(String::trim)
+                .collect(Collectors.toUnmodifiableList());
+        return carNameList;
+    }
```

Comment:
```suggestion
    public static List<String> getCarNameListFromCarNames(final String carNames) {
        return Arrays.stream(carNames.split(DELIMITER_CAR_NAMES))
                .map(String::trim)
                .toList();
    }
```
이렇게 줄일 수 있을 것 같아요!
toList() 메소드 안에 unmodifiableList도 포함되어있어서요~

Reply:
@skylar1220 헉.. 그렇군요!!! `toList()` 애용하겠습니다. ☺️